pkg/remediator/watch/manager.go

```diff
@@ -221,6 +222,10 @@ func (m *Manager) UpdateWatches(ctx context.Context, gvkMap map[schema.GroupVers
 		// Only start watcher if the resource exists.
 		// All resources should exist and be established by now. If not, error.
 		if _, err := m.mapper.RESTMapping(gvk.GroupKind(), gvk.Version); err != nil {
+			if meta.IsNoMatchError(err) {
```

Comment:
style nit: switch on error and then `continue` after the switch

Reply:
ya know, i think i tried that on the AddWatches last time and it popped me a linter error telling me to just use an if statement.

Reply:
but maybe i'm misremembering and two cases are ok but one case is bad.

Either way, if this were a return, you'd expect it like this. It just looks redundant with continue .

Reply:
This seems like good style IMO:
```golang
switch {
case meta.IsNoMatchError(err):
  errs = status.Append(errs, syncerclient.ConflictWatchResourceDoesNotExist(err, gvk))
default: 
  errs = status.Append(errs, status.APIServerErrorWrap(err))
}
continue
```

Reply:
done